contrib/benchmarking_nd4j/pom.xml

```diff
@@ -83,8 +88,8 @@
                 <version>3.8.1</version>
                 <configuration>
                     <compilerVersion>${javac.target}</compilerVersion>
-                    <source>${javac.target}</source>
-                    <target>${javac.target}</target>
```

Comment:
This looks a bit off, you bothch change the parameter as part of the properties, and then change it here to a different fixed value. I guess that isn't quite intentional?